Calcify/Basic.lean

```diff
@@ -59,12 +59,6 @@ def mkEqOfHEq' (h : Expr) : MetaM Expr := do
   | heq_of_eq _ _ _ h => pure h
   | _ => mkEqOfHEq h
 
-def mkHEqOfEq (h : Expr) : MetaM Expr := do
-  match_expr h with
-  | Eq.refl _ x => mkHEqRefl x
-  | eq_of_heq _ _ _ h => pure h
-  | _ => mkAppM ``heq_of_eq #[h]
-
```

Comment:
Note that this is an optimizing function, do likely not the same as in core. It should be renamed to have a prime and call the core one in the default case.

Reply:
Oops, didn't look at all, just wanted `lean-egg` to compile on nightly and this was the shortest path. I'll close.